rotkehlchen/types.py

```diff
@@ -67,6 +67,7 @@
 
 
 UNISWAP_PROTOCOL = 'UNI-V2'
+YEARN_VAULTS_V1_PROTOCOL = 'yearn-v1'
```

Comment:
why the difference in the naming convention? 

`YEARN_VAULTS_V1_PROTOCOL` uses a hyphen to separate words, `YEARN_VAULTS_V2_PROTOCOL` uses an underscore.
Perhaps just use underscore since v2 uses that already.

Also, I think you need to add it to `KnownProtocolAssets`.

Reply:
is not my choice. Was already in the database. The `KnownProtocolsAssets` tuple is there for protocols that we know how to calculate their price based in the protocol field (you can check this in the inquirer logic). We do this for V2 but not for V1. I'll add a comment

Reply:
Where is that comment? I am unresolving the issue as I was gonna ask the same question Isaac already asked a month ago :smile:

Reply:
Question: We have a global DB upgrade in this one release anyway. Why not just fix the discrepancy instead of accepting is as yet another "Technical debt". This upgrading can(should!) happen in another PR btw!

Reply:
Yes 100%. Will prepare the change and hang it to you

Reply:
I did that in the other PR. It's in the packaged global DB. So resolving this comment. The DB upgrade though has not happened.